src/public/packages/backpack/crud/css/list.css

```diff
@@ -241,4 +241,9 @@
 /*	Allow dropdown buttons to push the height/width of the rapper so that they're entirely visible	*/
 	.fixed .wrapper {
 		overflow: visible;
+	}
+
+	/* Reset button text display state inside modals */
+	.modal-dialog .btn span {
+		display: initial !important;
```

Comment:
This CSS rule resets the display inside the modal, developers can always override this, but by default the visibility is reset inside modals.